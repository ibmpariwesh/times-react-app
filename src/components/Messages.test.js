import { render } from "@testing-library/react"
import Messages from "./Messages"

test('first test for message component', ()=>{
    const component = render(<Messages message="message1"></Messages >).asFragment;
    expect(component()).toMatchSnapshot();
})

test('No message prop', ()=>{
    const component = render(<Messages></Messages >).asFragment;
    expect(component()).toMatchSnapshot();
})