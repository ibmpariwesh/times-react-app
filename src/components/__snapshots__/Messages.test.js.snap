// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No message prop 1`] = `
<DocumentFragment>
  <div
    class="hidden"
  />
</DocumentFragment>
`;

exports[`first test for message component 1`] = `
<DocumentFragment>
  <div
    class="alert alert-success"
  />
</DocumentFragment>
`;

exports[`first test for message component 2`] = `
<DocumentFragment>
  <div
    class="alert alert-success"
  />
</DocumentFragment>
`;
